### ISSUE TITLE
Auto-generated code issue of parameters' sequence

### ISSUE THREAD

**Issue by keval-d** [vitusortner/floor#399]
This is Dao fun
```
@Query('UPDATE ShareApp SET isPresent = :isPresent WHERE appId = :appId')
  Future<void> updateShareApp(String appId, bool isPresent);
```

this is auto generated code
```
@override
  Future<void> updateShareApp(String appId, bool isPresent) async {
    await _queryAdapter.queryNoReturn(
        'UPDATE ShareApp SET isPresent = ? WHERE appId = ?',
        arguments: <dynamic>[
          appId,
          isPresent == null ? null : (isPresent ? 1 : 0)
        ]);
  }
```
It's taking fun parameters' sequence not in **Query sequence**

**Comment by mqus** [comment]
This issue is known(#224) and will be solved with #361.

**Comment by mqus** [comment]
This issue was solved with #509
